application/templates/static_site/measure.html

```diff
@@ -100,9 +100,35 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <h1 class="govuk-heading-l" itemprop="headline">
+      <h1 class="govuk-heading-xl govuk-!-margin-bottom-4  govuk-!-margin-top-2" itemprop="headline">
         {{ measure_version.title }}
       </h1>
+
+      {%  if measure_version.first_published_date %}
+        <p class="govuk-body">Published <time itemprop="datePublished" datetime="{{ measure_version.first_published_date | format_iso8601_date }}">{{ measure_version.first_published_date | format_friendly_date }}</time>
+        </p>
+      {%  endif %}
+
+      <div class="govuk-body govuk-!-font-size-16 govuk-!-margin-bottom-1 govuk-!-margin-top-5">Contents</div>
+      <ol class="govuk-list eff-list__table-of-contents govuk-!-margin-bottom-9">
+        <li><a class="govuk-link" href="#main-facts-figures" data-on="click" data-event-category="Table of contents link clicked" data-event-action="Main facts and figures" data-event-label=""><span class="eff-list__table-of-contents__number">1. </span><span class="eff-list__table-of-contents__content">Main facts and figures</span></a></li>
+
+
+        {% if measure_version.dimensions %}
```

Comment:
I don't think we should need this `if` condition. `measure_version.dimensions` should always exist, but just as an empty list if there aren't any. Which is fine based on the HTML that follows.